packages/web-twig/src/Twig/PropsExtension.php

```diff
@@ -29,6 +29,11 @@ public function getFunctions(): array
                 'needs_environment' => true,
                 'is_safe' => ['html'],
             ]),
+            new TwigFunction('styleProp', [$this, 'renderStyleProp'], [
+                'needs_environment' => true,
+                'is_safe' => ['html'],
```

Comment:
This option disables escaping, see https://twig.symfony.com/doc/2.x/advanced.html#automatic-escaping

Please remove this line and add some examples with custom styles to ensure that everything is safely rendered. Thanks.

For more information see our discussion on Slack: https://almamedia.slack.com/archives/C0502FGLG3G/p1681476431357859 or contact @jirinovak

Reply:
https://slack-files.com/T0325RBAD-F053G5D059T-2af92bb610 I have to keep it here as it breaks quotes in the embedded twig file.